test/parallel/test-tls-key-mismatch.js

```diff
@@ -32,8 +33,8 @@ const errorMessageRegex =
   /^Error: error:0B080074:x509 certificate routines:X509_check_private_key:key values mismatch$/;
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent2-cert.pem`)
+  key: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-key.pem`),
```

Comment:
This could be using method `fixtures.readKey` instead of `fs.readFileSync` to get the keys.

Reply:
@pawelgolda Sure, but `fixtures.readKey` also requires an `enc`, not sure what to pass in there 🤔

Reply:
You can leave that param out empty.

Reply:
Nevermind, I see from the documentation that `enc` is indeed optional, so I will actually go ahead and make that change.

Reply:
@pawelgolda Doing that change breaks the test for some reason..

Reply:
@kasimdoctor Can you post the code?

Reply:
It should definitely just be...

```js
key: fixtures.readKey('agent1-key.pem'),
cert: fixtures.readKey('agent1-cert.pem')
```

Reply:
@jasnell I get the following error when running the test with the code change you requested:

```shell
=== release test-tls-key-mismatch ===
Path: parallel/test-tls-key-mismatch
assert.js:45
  throw new errors.AssertionError({
  ^

AssertionError [ERR_ASSERTION]: Missing expected exception.
    at innerThrows (assert.js:683:7)
    at Function.throws (assert.js:702:3)
    at Object.<anonymous> (/src/repos/personal/node/test/parallel/test-tls-key-mismatch.js:41:8)
    at Module._compile (module.js:599:30)
    at Object.Module._extensions..js (module.js:610:10)
    at Module.load (module.js:518:32)
    at tryModuleLoad (module.js:481:12)
    at Function.Module._load (module.js:473:3)
    at Function.Module.runMain (module.js:640:10)
    at startup (bootstrap_node.js:187:16)
Command: out/Release/node /src/repos/personal/node/test/parallel/test-tls-key-mismatch.js
```

Reply:
@kasimdoctor make sure to use `'agent2-cert.pem'` for the cert.

Reply:
@lpinca Thanks! That was a dumb mistake on my part 😄